vllm/attention/backends/ipex_attn.py

```diff
@@ -456,59 +486,81 @@ def forward(
             # TODO(woosuk): Tune this heuristic.
             # For context len > 8192, use V2 kernel to avoid shared memory
             # shortage.
-            use_v1 = (max_seq_len <= 8192 and
-                      (max_num_partitions == 1 or num_seqs * num_heads > 512))
-            if use_v1:
-                # Run PagedAttention V1.
-                ipex_ops.paged_attention_v1(
+
+            bsz = len(decode_meta.seq_lens)
+            import vllm._C.ops
+            if using_gqa_kernel:
+                block_size = value_cache.shape[2]
+                vllm._C.ops.paged_attention_gqa(
                     out,
                     decode_query,
                     key_cache,
                     value_cache,
+                    bsz,
+                    self.num_heads,
                     self.num_kv_heads,
                     self.scale,
                     decode_meta.block_tables,
                     decode_meta.seq_lens_tensor,
                     block_size,
-                    max_seq_len,
-                    self.alibi_slopes,
```

Comment:
Seems current gqa kernel doesn't support alibi_slopes. Should be fixed.

Reply:
We can mark alibi as todo in current stage.

Reply:
ok